model_specific/SM/standard_model.cpp

```diff
@@ -625,9 +625,14 @@ void Standard_model::calculate_pole_masses()
    calculate_MFd_pole();
    calculate_MFu_pole();
    calculate_MFe_pole();
-   calculate_MVWp_pole();
 
 #endif
+   double mt_pole = qedqcd.displayPoleMt();
```

Comment:
If you don't mind, please move these lines into a separate function, maybe called `calculate_MVWp_pole_fit`.

Reply:
Done in 3923909b5e2ead6a8a570a94fd6bc8b94f6d62b6.